src/index.html

```diff
@@ -1,5 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
+
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css">
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css">
+  <title>freeCodeCamp Manila</title>
+</head>
+
+<body>
+  <!-- hero section -->
+  <div class="container">
+    <h1>Learn to Code with Us!</h1>
+    <p>We are a local community and study group learning to code through freeCodeCamp.org</p>
+    <a href="https://www.freecodecamp.org/" target="_blank" class="button">Join the community</a>
+    <div class="socilamediaMileston">
+      <a href="https://www.facebook.com/groups/free.code.camp.manila/" target="_blank"><i
+          class="fab fa-facebook-square fa-2x"></i></a>
+      <span>4000+ Members, 1000+ Likes</span>
+    </div>
+    <img id="hero-image" src="images/hero.jpg" alt="">
+  </div>
+</body>
```

Comment:
You also have duplicated `body` tag, add the hero section before the Join the Community section